### PR TITLE
Follow up changes to PR 526

### PR DESCRIPTION
**Pull request by cblecker**
This contains follow up changes to #526 

https://github.com/kubernetes/k8s.io/pull/526#pullrequestreview-339581142

/assign @thockin

### CODE REVIEW

File: infra/gcp/ensure-main-project.sh [hunk 1]
```diff
@@ -129,8 +129,12 @@ gcloud projects add-iam-policy-binding "${PROJECT}" \
 # Bootstrap DNS zones
 ensure_dns_zone "${PROJECT}" "k8s-io" "k8s.io"
 ensure_dns_zone "${PROJECT}" "kubernetes-io" "kubernetes.io"
+ensure_dns_zone "${PROJECT}" "x-k8s-io" "x.k8s.io"
```
Comment (thockin) [inline infra/gcp/ensure-main-project.sh]: Shouldn't this be x-k8s ?

File: infra/gcp/ensure-main-project.sh [hunk 2]
```diff
@@ -129,8 +129,12 @@ gcloud projects add-iam-policy-binding "${PROJECT}" \
...
+ensure_dns_zone "${PROJECT}" "k8s-e2e-com" "k8s-e2e.com"
 ensure_dns_zone "${PROJECT}" "canary-k8s-io" "canary.k8s.io"
 ensure_dns_zone "${PROJECT}" "canary-kubernetes-io" "canary.kubernetes.io"
+ensure_dns_zone "${PROJECT}" "canary-x-k8s-io" "canary.x.k8s.io"
```
Comment (thockin) [inline infra/gcp/ensure-main-project.sh]: Same

File: infra/gcp/ensure-main-project.sh (hunk 1 above)
Comment (cblecker) [reply infra/gcp/ensure-main-project.sh]: Fixed.

File: infra/gcp/ensure-main-project.sh (hunk 2 above)
Comment (cblecker) [reply infra/gcp/ensure-main-project.sh]: Fixed.

**Pull request merged**